### PR TITLE
MON-3524: Update metrics-server in CHANGELOG and README

### PR DESCRIPTION
**Pull request by slashpai**
* [ ] I added CHANGELOG entry for this change.
* [x] No user facing changes, so no entry in CHANGELOG was needed.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Note: This CHANGELOG is only for the monitoring team to track all monitoring related changes. Please see OpenShift release notes for official changes.
 
+## 4.15
+
+- [#2022](https://github.com/openshift/cluster-monitoring-operator/pull/2022)
+Add support to switch to metrics server from prometheus-adapter in TechPreview mode
```
Comment (simonpasquier) [inline CHANGELOG.md:6]:
```suggestion
Add support to switch to metrics server from prometheus-adapter when the `MetricsServer` feature gate is enabled.
```

**Comment by slashpai** [comment]
@simonpasquier please review again

**Pull request merged**